test/unit/reducers/clear_user_data_test.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:crowdleague/actions/auth/clear_user_data.dart';
+import 'package:crowdleague/models/app/app_state.dart';
+import 'package:crowdleague/reducers/app_reducer.dart';
+import 'package:redux/redux.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('ClearUserDataReducer', () {
+    test('Clears any user data from state', () {
+      // Create a basic store with the app reducers
+      final store = Store<AppState>(
+        appReducer,
```

Comment:
I imagine I've been doing this myself (using the appReducer) but I think we should use just the reducer under test, if possible.  I'll see if I can make the change and add it to the PR so we can discuss tomorrow...

Reply:
I tried testing the reducer directly without the store and it seems to work nicely.  I put up #251 as an example. After discussing with @tamari-gray he's going to change the other test to use this method.

Reply:
discussed :)